zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```diff
@@ -578,6 +578,45 @@ paths:
             application/problem+json:
               schema:
                 $ref: "#/components/schemas/ProblemDetail"
+
+  /administration/clock/pin:
+    post:
+      tags:
+        - Administration
+        - Clock control
+      summary: Pin the Zeebe engine’s internal clock to a specific time (experimental)
+      description: |
+        Set a precise, static time for the Zeebe engine’s internal clock. When the clock is pinned,
+        it remains at the specified time and does not advance. To change the time, the clock must be
+        pinned again with a new timestamp.
+
+        :::note
+        This endpoint is experimental. It may undergo changes or improvements in future releases.
+        :::
+      operationId: pinClock
```

Comment:
❓ Is this required ? I think it can be removed since we are not linking operations

Reply:
👍 You're right, it's probably not necessary at this point. I've removed it. If we need it in the future, we can add it back in a centralized manner through a separate PR.